src/main/java/com/aetherteam/aether_genesis/block/container/BlockAetherCraftingTable.java

```diff
@@ -0,0 +1,44 @@
+package com.aetherteam.aether_genesis.block.container;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.stats.Stats;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.MenuProvider;
+import net.minecraft.world.SimpleMenuProvider;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.inventory.ContainerLevelAccess;
+import net.minecraft.world.inventory.CraftingMenu;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.CraftingTableBlock;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+import org.jetbrains.annotations.NotNull;
+
+public class BlockAetherCraftingTable extends CraftingTableBlock
```

Comment:
Is overriding use() and getMenuProvider() necessary in this class? If not, couldn't you also just do away with this class entirely and register the Skyroot Crafting Table with the default CraftingTableBlock?